docs/first-migration.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Track your first Migration
+sidebar_label: Track your first Migration
+---
+
+Migrations give you the ability to track the completion of a migration within the effx platform.  Common use cases are:
+  - upgrading all of your microservices to a particular language version
+  - moving all of your microservices from one platform to another
+
+## Defining your first Migration
+
+A migration is comprised of the following parts
+- `Name` - this is the identifier of the migration that your users will see
+- `Description` - this is the purpose, incentive, or value of the migration
+- `Timeline` - track the Start and End date of the migration so that service owners will know when it's needed to be completed
+- `Email` - this is the e-mail address of who to reach out to for more information about the migration
+- `Link` - provide external links for more information about the migration.  For example, the link to instructions on how to complete
+
+### Migration query defined
+
+The migration query is key to measuring the status of the migration.  There are two ways of defining the query:
+`Query by value`: Use this type of query when you want all of your tags to eventually be equal to a particular value.  For example, if you're moving all of your `deployment` (key) to `kubernetes` (value), you could define your Query as such:
+
+![Query by value](/img/migration-kubernetes.png)
+
+`Query by version`: Use this type of query when you want to compare semantic version tags.  For example, if you're wanting to upgrade all of your `go` services to `1.15.0` or greater, you can use this type of query:
+
+![Query by value](/img/migration-go.png) 
```

Comment:
The spacing here was condensed when I viewed this locally. I think adding some `<br>` will help.